plugins/release.js

```diff
@@ -62,13 +71,19 @@ class WandbPlugin extends Plugin {
         since: publishedAt,
       });
       //new Date(commit.author.date) > new Date(publishedAt)
-      if(res.data.length > 100) {
-        console.warn("There have been more than 100 commits since the last release!")
+      if (res.data.length > 100) {
+        console.warn(
+          'There have been more than 100 commits since the last release!'
+        );
       }
       const notes = res.data
         .map((commit) => `* ${commit.commit.message.split('\n')[0]}`)
         .join('\n');
-      this.setContext({notes})
+      this.setContext({notes});
+
+      this.setContext({
+        lastCommitInRelease: res.data[res.data.length - 1].commit.tree.sha,
```

Comment:
This looks great.  An epic bonus would be to show the user the 5 most recent commits and let them choose which one to release from.  This makes it more safe for cases where we merge our release branch into master and new changes make it in before we get around to cutting the release.